Tanvir04Khan/01-max-udemy/04-food-order-app/src/components/Header.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Fragment } from "react";
+
+import tableFullOfFastFood from "../assets/fast-food-table.jpg";
+import locateMeSymbol from "../assets/location.png";
+
+const Header = (props) => {
```

Comment:
I dont see any props used here. 
and
any any props passed on to this component